source/expectations/data_values/ut_compound_data_value.tpb

```diff
@@ -229,63 +251,80 @@ create or replace type body ut_compound_data_value as
     l_column_filter := ut_compound_data_helper.get_columns_filter(a_exclude_xpath, a_include_xpath);
       
     /**
-    * Due to incompatibility issues in XML between 11 and 12.2 and 12.1 versions we will prepopulate pk_hash upfront to
-    * avoid optimizer incorrectly rewrite and causing NULL error or ORA-600
-    **/        
+    * Due to incompatibility issues in XML between 11 and 12.2 and 12.1 versions we will prepopulate item_hash upfront to
+    * avoid optimizer incorrectly rewrite and causing NULL error or ORA-600.
+    * We will also update item_data by include exclude values so we dont have to apply transformation over and over later.
+    **/     
     execute immediate 'merge into ' || l_ut_owner || '.ut_compound_data_tmp tgt
                        using (
                               select '||l_ut_owner ||'.ut_compound_data_helper.get_hash(ucd.item_data.getclobval()) item_hash, 
-                                      pk_hash, ucd.item_no, ucd.data_id
+                                      ucd.item_no, ucd.data_id, ucd.item_data
                               from
                               (
-                              select '||l_column_filter||','||get_column_pk_hash(a_join_by_xpath)||', item_no, data_id
+                              select '||l_column_filter||', item_no, data_id
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: An illegal character with code point 10 was found in this literal. ([plsql:LiteralsNonPrintableCharactersCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3ALiteralsNonPrintableCharactersCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=760&issues=AWY_N4wenzfF1D4JsH2J&open=AWY_N4wenzfF1D4JsH2J)

[ISSUE_KEY]: AWY_N4wenzfF1D4JsH2J